src/main/kotlin/com/karumi/maxibonkata/Chat.kt

```diff
@@ -0,0 +1,11 @@
+package com.karumi.maxibonkata
+
+interface Chat {
+    fun sendMessage(message: String)
+}
```

Comment:
I've added this interface even when it's not needed because we have a similar implementation in our katas and it's used during the training to explain why to use interfaces and why not. If you think it's just adding noise I can remove it.

Reply:
As long as the main purpose of this piece of code is to be didactic, I think it worth to have this interface.